ui/raidboss/data/triggers/unending_coil_ultimate.js

```diff
@@ -1240,9 +1361,10 @@
         'Ragnarok': 'Ragnarök',
         'Thunderwing': 'Donnerschwinge',
         'Twintania': 'Twintania',
-        // FIXME:
-        'Tail of Darkness': 'Tail of Darkness',
-        'Fang of Light': 'Fang of Light',
+        // FIXED:
+        'Tail of Darkness': 'Dunkelschweif',
+        'Fang of Light': 'Lichtklaue',
+        // HAVE TO CONFIRM WHEN HOME:
```

Comment:
I think there are a bunch of places that will need to get changed (including some scripts) if this is the wrong Engage! translation.